apps/io-services-cms-webapp/src/utils/converters/service-lifecycle-converters.ts

```diff
@@ -76,7 +76,9 @@ export const itemToResponse =
         (err) => ResponseErrorInternal(err.message),
         (topic) => ({
           id,
-          last_update: last_update ?? new Date().getTime().toString(),
+          last_update: modified_at
+            ? DateUtils.isoStringfromUnixTimestamp(modified_at)
+            : new Date().getTime().toString(),
```

Comment:
chore: fix `toISOString()` instead of `toString()`
```suggestion
            : new Date().getTime().toISOString(),
```

Reply:
Should be this `new Date().toISOString()`